ironfish/src/blockchain/database/blockchaindb.ts

```diff
@@ -422,6 +432,18 @@ export class BlockchainDB extends TransactionalDatabase {
     return this.ethTransactionHashToTransactionHash.del(ethTransactionHash, tx)
   }
 
+  async putEvmReceipt(
+    ethTransactionHash: Buffer,
+    receipt: EvmReceiptValue,
+    tx?: IDatabaseTransaction,
+  ): Promise<void> {
+    return this.evmTransactionReceipts.put(ethTransactionHash, receipt, tx)
+  }
+
```

Comment:
Gonna need a get field, might as well add it here

Reply:
Added!